SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -46,6 +46,16 @@ SequenceableCollection : Collection {
 		});
 		^obj
 	}
+	*recursiveFormula { arg a, size;
+		var i=0, divisor = 3, divisorDec = 0.9;
+		var obj = this.new(size);
+		while { i < size }{
+			obj.multiply((a/divisor));
```

Comment:
This does nothing.
1. There is no method `multiply` for multiplication. Use `*` instead.
2. `obj` will be empty (only reserved for a certain size). Multiplying nothing by some value is still nothing.